src/applications/coronavirus-vaccination/components/Unsubscribe.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect, useState } from 'react';
+import { connect } from 'react-redux';
+import { withRouter } from 'react-router';
+import { requestStates } from 'platform/utilities/constants';
+import LoadingIndicator from '@department-of-veterans-affairs/formation-react/LoadingIndicator';
+
+import { unsubscribe } from '../api';
+
+import { focusElement } from 'platform/utilities/ui';
+
+// Assumes URL is like /unsubscribe?sid={sid}
+function Unsubscribe({ router }) {
+  const subscriberId = router.location.query.sid;
+
+  const [unsubscribeStatus, setUnsubscribeStatus] = useState(
+    requestStates.notCalled,
+  );
+
+  useEffect(
+    () => {
+      focusElement('#covid-vaccination-heading-unsubscribe');
+      async function unsubscribeBySid() {
+        try {
+          await unsubscribe(subscriberId);
+          setUnsubscribeStatus(requestStates.succeeded);
+        } catch (error) {
+          setUnsubscribeStatus(requestStates.failed);
+        }
+      }
+      if (subscriberId !== undefined) unsubscribeBySid();
+      else setUnsubscribeStatus(requestStates.failed);
+    },
+    [subscriberId],
+  );
+
+  let content = <LoadingIndicator message="Unsubscribing..." />;
+
+  if (unsubscribeStatus === requestStates.failed) {
+    content = (
+      <div className="va-introtext">
+        <p>Sorry, we were not able to unsubscribe you at this time.</p>
+      </div>
+    );
+  } else if (unsubscribeStatus === requestStates.succeeded) {
+    content = (
+      <div className="va-introtext">
+        <p>You have been unsubscribed from updates.</p>
+      </div>
+    );
+  }
+
+  return (
+    <>
+      <h1> Unsubscribe </h1>
```

Comment:
```suggestion
      <h1 id="covid-vaccination-heading-unsubscribe"> Unsubscribe </h1>
```

My fault for posting incomplete code!